python-auto-metadata-collection/README.md

```diff
@@ -0,0 +1,49 @@
+# Machine Learning Metadata Extraction Library
+This library provides a unified way to extract metadata from machine learning models built using Scikit-learn, TensorFlow/Keras, and PyTorch. It also includes a base API client to facilitate interaction with AI4HF Passport Server. Extracted metadata can sent to AI4HF Passport Server.
```

Comment:
Let's change 'Extracted metadata can sent to AI4HF Passport Server.' to 'Extracted metadata can be sent to AI4HF Passport Server.'